models/staging/stg_trust_lines.yml

```diff
@@ -0,0 +1,125 @@
+version: 2
+
+models:
+  - name: stg_trust_lines
+    tests:
+      - dbt_utils.recency:
+          datepart: day
+          field: cast(closed_at as timestamp)
+          interval: 2
+          meta:
+            description: "Monitors the freshness of your table over time, as the expected time between data updates."
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - account_id
+            - asset_code
+            - asset_issuer
+            - liquidity_pool_id
```

Comment:
nit: I _think_ you could use `ledger_key` instead of these three columns

Reply:
This looks like an expensive test and taking a while to run on 86B records. Wondering should run these tests only on newer data?

Reply:
+1 agree. Thank you for catching this. Generally, these tests will be expensive on state tables. This being one of the more expensive tests because of table size. Let's just run on newer data, almost in incremental mode until we come up with a better solution.